api/server/routes/config.js

```diff
@@ -1,5 +1,6 @@
 const express = require('express');
 const router = express.Router();
+const packageJson = require('../../../package.json');
```

Comment:
this is the issue. we don't want to require the package file at all.

Reply:
This was my initial understanding. When I asked GPT to confirm it, it told me: "By making this change, the version number will not be directly exposed in the frontend code and will be retrieved securely from the backend server."

Reply:
*I pushed an update where it's directly written in the footer because it seemed like the easiest alt method to maintain